share/availability/light/availability.go

```diff
@@ -64,7 +64,7 @@ func NewShareAvailability(
 		getter:        getter,
 		bs:            bs,
 		params:        params,
-		storageWindow: availability.StorageWindow,
+		storageWindow: availability.RequestWindow,
```

Comment:
Should be availability.Sampling window? Also would be nice to rename storageWindow to sampling window, since it controls what will be sampled

Reply:
done https://github.com/celestiaorg/celestia-node/pull/4427/commits/ac299040daf48f29cf59436a5f22c1bff7808a41